tools/c7n_azure/c7n_azure/resources/appserviceplan.py

```diff
@@ -143,14 +144,13 @@ def _process_resource(self, resource):
         if resource['kind'] == 'linux':
             model.reserved = True
 
-        if 'size' in self.data:
-            size = Lookup.extract(self.data.get('size'), resource)
-            model.sku = models.SkuDescription()
-            model.sku.tier = ResizePlan.get_sku_name(size)
-            model.sku.name = size
+        size = Lookup.extract(self.data.get('size'), resource)
+        model.sku = models.SkuDescription()
+        model.sku.tier = ResizePlan.get_sku_name(size)
+        model.sku.name = size
```

Comment:
Should these be properties.targetWorkerSizeId and properties.workerTierName?

https://docs.microsoft.com/en-us/rest/api/appservice/appserviceplans/update#request-body

Reply:
Same issue as described [here](https://github.com/cloud-custodian/cloud-custodian/pull/4735#discussion_r321593790)